### PR TITLE
fix: clients created in a group

### LINKED ISSUE #779: Adding Client In a group Issue
Hello Devs.
There is an Issue I noticed When you Create a client in  group using the App, The created client appears not to be belonging to any group ie, The App has no ability to manage client creation in a group.

Also I request a feature to manage(ie, add already created clients) clients to a group and Manage(ie, add already created groups) groups in Centers. 
Thanks

### PR DESCRIPTION
**Pull request by laxyapahuja**
Fixes #779 

- [x] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [x] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by laxyapahuja** [comment]
@luckyman20 @eliherema @santoshconflux @moksh-mahajan Can you please review this task? Thanks.

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/online/createnewclient/CreateNewClientFragment.java [hunk 1]
```diff
@@ -147,6 +149,11 @@ public void onCreate(Bundle savedInstanceState) {
         clientTypeList = new ArrayList<>();
         officeList = new ArrayList<>();
         staffList = new ArrayList<>();
+        Bundle bundle = this.getArguments();
+        if (bundle != null) {
```
Comment (luckyman20) [inline mifosng-android/src/main/java/com/mifos/mifosxdroid/online/createnewclient/CreateNewClientFragment.java]:
```suggestion
        if (bundle) {
```

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/online/createnewclient/CreateNewClientFragment.java [hunk 2]
```diff
@@ -147,6 +149,11 @@ public void onCreate(Bundle savedInstanceState) {
...
+            int groupId = getArguments().getInt("groupId");
+            mGroupId = groupId;
```
Comment (luckyman20) [inline mifosng-android/src/main/java/com/mifos/mifosxdroid/online/createnewclient/CreateNewClientFragment.java]: Assign it directly, why are you making a new variable?

File: mifosng-android/src/main/java/com/mifos/api/services/GroupService.java [hunk 3]
```diff
@@ -63,4 +64,9 @@
     Observable<GenericResponse> activateGroup(@Path("groupId") int groupId,
                                                @Body ActivatePayload activatePayload);
 
+    @POST(APIEndPoint.GROUPS + "/{groupId}?command=associateClients")
+    Observable<GenericResponse> associateClients(@Path("groupId")int groupId,
```
Comment (luckyman20) [inline mifosng-android/src/main/java/com/mifos/api/services/GroupService.java]:
```suggestion
    Observable<GenericResponse> associateClients(@Path("groupId") int groupId,
```

**Comment by Aditya-gupta99** [comment]
We are closing this PR as it causes a branch conflict since the Java file has now been migrated to kotlin

**Pull request closed without merging**